owners/src/notifier.js

```diff
@@ -100,11 +100,24 @@ class OwnersNotifier {
       return;
     }
 
-    const comment = fileNotifyComments.join('\n\n');
+    const body = fileNotifyComments.join('\n\n');
+    let comment;
     if (botComment) {
-      await github.updateComment(botComment.id, comment);
+      comment = await github.updateComment(botComment.id, body);
     } else {
-      await github.createBotComment(this.pr.number, comment);
+      botComment = await github.createBotComment(this.pr.number, body);
+    }
+
+    // App-authenticated comments appear to come from the app user (ie.
+    // `amp-owners-bot`) but can't tag teams. The workaround is doing a fake
+    // update to the comment with a user-authenticated client, which enables the
+    // @ mention but leaves the comment author as the app name.
+    const teamNotifies = Object.keys(notifies).filter(n => n.includes('/'));
+    if (teamNotifies.length) {
+      await github.user.updateComment(
+        botComment.id,
+        `${body}\n<!-- Edited to fix team @mention -->`
```

Comment:
`@mention` → `@ mentions` just in case the parser decides to tag the account called `mention` one day (this account doesn't exist, but we should be nice)

Reply:
Good catch! I usually do that when typing "@ mention", missed it here!

Reply:
(Though this _is_ a comment, so it wouldn't tag anyway)